api/controllers/v1/location/get-snapshots.js

```diff
@@ -0,0 +1,54 @@
+const ControllerService = require('../../../services/ControllerService');
+const ErrorService = require('../../../services/ErrorService');
+const RightService = require('../../../services/RightService');
+
+module.exports = async (req, res) => {
```

Comment:
Hello @bsoufflet, here is the management of the sensitive entrance. 
Can you have a look on it and tell us if you have any comments ?

Reply:
Plutot que de faire ce check ici ne faudrait il pas mieux améliorer `remove-sensitive-entrances.js` ?
@Clm-Roig tu en penses quoi ?